api/shared_collection.py

```diff
@@ -45,16 +41,15 @@ def __init__(self, _db, library, api_map=None):
            calls, we only instantiate one CirculationAPI per library,
            and keep them around as long as possible.
```

Comment:
This is copy-and-pasted. I believe the right thing to say is that we only instantiate one SharedCollectionAPI and try to keep it around as long as possible.